themes/reachout/composer.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "mautic/theme-reachout",
+  "description": "Mautic ReachOut Theme",
+  "type": "mautic-theme",
+  "keywords": ["mautic","theme"],
+  "minimum-stability": "dev",
+  "require": {
+    "mautic/core-lib": ">=5.2"
```

```suggestion
    "mautic/core-lib": ">=6.0"
```

Shouldn't this be 6.0? Or is 5.2.0 the minimum?